docs/blog/newsletter-may-2020.rst

```diff
@@ -0,0 +1,91 @@
+.. post:: May 04, 2020
+   :tags: newsletter
+
+####################################
+Write the Docs Newsletter – May 2020
+####################################
+
+Hello, documentarians; it's Beth here with the May edition of the Write the Docs newsletter. I hope you're all holding up okay under lockdown.
+
+We just announced last week that we're cancelling our on-site conferences in Portland and Prague, and moving online. Virtual Portland will be in August, with exact dates to be announced soon; and virtual Prague will be in the autman, with dates announced in summer. There's a lot more detail in`the blog post </blog/moving-portland-prague-2020-online/>`__, including what to do if you've already got a Portland ticket.
+
+To help you entertain yourself between now and August, there's some good news from the Write the Docs website. We've got an extensive archive of conference talks and newsletter articles, covering all sorts of topics, but up until now it's been a bit difficult to browse them all. So I'm delighted to introduce the the `WTD content index </topics/>`__. If there's a topic you'd like to spend some time learning about, check out the index and see if your topic is there!
```

Comment:
Write The Docs, right?

Reply:
This might be a good spot to mention again clicking the newsletter icon to add things in for newsletter consideration if their topic isn't in there, so that it one day might be.

Reply:
Correct, thanks! And I've added in a note about the icon.